worlds/sc2/mission_tables.py

```diff
@@ -297,6 +297,13 @@ class MissionInfo(NamedTuple):
         short_name = mission.mission_name[:mission.mission_name.find(' (')]
         lookup_name_to_mission[short_name] = mission
 
+lookup_short_name_to_mission_variants: Dict[str, Set[SC2Mission]] = {
+    mission.mission_name[:mission.mission_name.find(' (')]: {mission} for mission in SC2Mission if mission.flags & MissionFlag.HasRaceSwap
+}
+for mission in SC2Mission:
+    if mission.flags & MissionFlag.RaceSwap:
+        lookup_short_name_to_mission_variants[mission.mission_name[:mission.mission_name.find(' (')]].add(mission)
```

Comment:
Why not just automatically generate the mission groups in mission_groups.py? Pretty sure the option already looks at those.

Reply:
I'm not terribly familiar with how mission groups work, so I didn't know that was an option! I'll give that a try instead.

Reply:
Done, much cleaner this way.